src/plugins/foxPage/foxPage.tsx

```diff
@@ -1,14 +1,86 @@
-import { RawText } from 'components/Text'
+import { SimpleGrid, TabList, Tabs } from '@chakra-ui/react'
+import { AssetId } from '@shapeshiftoss/caip'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
 
-export enum FoxPageTab {
-  Fox = 'Fox',
-  Foxy = 'Foxy',
-  OneFox = 'OneFox',
+import { FoxTab } from './components/FoxTab'
+import { Layout } from './components/Layout'
+import { Total } from './components/Total'
+
+export enum FoxPageRoutes {
+  Fox = '/fox/fox',
+  Foxy = '/fox/foxy',
+}
+
+export const FoxAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
+export const FoxyAssetId = 'eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3'
+export const OneFoxAssetId = 'eip155:1/erc20:0x03352d267951e96c6f7235037c5dfd2ab1466232'
+
+export type FoxPageProps = {
+  activeAssetId: AssetId
 }
 
-export const FoxPage = (props: any) => {
-  // TODO remove console log when implementing tabs
-  // eslint-disable-next-line no-console
-  console.log(props)
-  return <RawText>TODO Fox Page</RawText>
+export const FoxPage = (props: FoxPageProps) => {
+  const translate = useTranslate()
+  const history = useHistory()
+  const assetFox = useAppSelector(state => selectAssetById(state, FoxAssetId))
+  const assetFoxy = useAppSelector(state => selectAssetById(state, FoxyAssetId))
+  const assetOneFox = useAppSelector(state => selectAssetById(state, OneFoxAssetId))
+  const foxTabSelected = props.activeAssetId === FoxAssetId
+  const foxyTabSelected = props.activeAssetId === FoxyAssetId
+
+  const handleClickFoxTab = () => {
+    if (foxTabSelected) {
+      return
+    }
+    history.push(FoxPageRoutes.Fox)
+  }
+
+  const handleClickFoxyTab = () => {
```

Comment:
I would say, since we are expecting only one more tab for the moment (oneFox), we are fine with the current solution as you said. At one point we may want to implement the solution you are highlighting of course.